src/annotation.js

```diff
@@ -876,6 +1306,84 @@ var rectangleAnnotation = function (args) {
     }
   };
 
+  /**
+   * Process any edit actions for this annotation.
+   *
+   * @param {geo.event} evt The action event.
+   * @returns {boolean|string} `true` to update the annotation, falsy to not
```

Comment:
'true' --> Truthy to update the annotation? To keep it consistent with other docs?

Reply:
On return values, we can be specific -- it *does* output `true`, not just an arbitrary truthy value.  On input parameters we try to be flexible and accept any truthy value.